### ISSUE TITLE
Broken dependency monii/container-interop-laravel

### ISSUE THREAD

**Issue by morrizon** [prooph/laravel-package#12]
Hi,

the dependency monii/container-interop-laravel doesn't exist anymore. Maybe the account was removed. I just uploaded the cache of my vendor here[1] (maybe you need it).

The error:

```
  - Installing monii/container-interop-laravel (v1.0.1)
    Downloading: Failed
    Failed to download monii/container-interop-laravel from dist: The "https://api.github.com/repos/monii/monii-container-interop-laravel/zipball/fe07223add49399c4384f1444b0b3507458a466d" file could not be downloaded (HTTP/1.1 404 Not Found)
    Now trying to download from source
  - Installing monii/container-interop-laravel (v1.0.1)
    Cloning fe07223add49399c4384f1444b0b3507458a466d


  [RuntimeException]
  Failed to clone https://github.com/monii/monii-container-interop-laravel.git via https, ssh protocols, aborting.
  - https://github.com/monii/monii-container-interop-laravel.git
    Cloning into '/home/vagrant/src/base-web-app/vendor/monii/container-interop-laravel'...
    remote: Invalid username or password.
    fatal: Authentication failed for 'https://github.com/monii/monii-container-interop-laravel.git/'
  - git@github.com:monii/monii-container-interop-laravel.git
    Cloning into '/home/vagrant/src/base-web-app/vendor/monii/container-interop-laravel'...
    ERROR: Repository not found.
    fatal: Could not read from remote repository.

    Please make sure you have the correct access rights
    and the repository exists.
```

[1] https://github.com/morrizon/container-interop-laravel

**Comment by morrizon** [comment]
I found a clone of the project[1] and the author or collaborator[2]

[1] https://github.com/bweston92/monii-container-interop-laravel
[2] https://github.com/simensen

**Comment by sandrokeil** [comment]
@morrizon We should get rid of this package. See [this comment](https://github.com/prooph/laravel-package/pull/1#issuecomment-178617825). Can you provide a container interop PR to Laravel, please?

**Comment by bweston92** [comment]
@sandrokeil I don't know how easy it would be to get it into the Laravel core or how long it would take? Maybe worth putting it onto prooph for now to keep this package working.

**Comment by codeliner** [comment]
@bweston92 We could add our own laravel container wrapper to the package so that we don't need to rely on an external package. It is one class with a few LoC. Maybe the simplest approach atm?

**Comment by sandrokeil** [comment]
I would prefer that this is solved via Laravel core. Taylor Otwell said:

> I don't have a big problem with it if someone wants to send over a PR.

/cc @morrizon

**Comment by bweston92** [comment]
I agree with Laravel having this at their core but for now no one can
install this package afaik.

**Comment by codeliner** [comment]
@sandrokeil I get your point. But I don't like it that we promote Laravel support on getprooph.org but have this broken package here. That's really bad! And the only thing that stops us atm is this simple container wrapper. I'd say add it to the package and let the Laravel package work again.
Then we can continue with more interesting stuff like new prooph versions support and some unique Laravel features like static facades.

**Comment by codeliner** [comment]
Fixed with #14
